plugins/plugin-postcss/plugin.js

```diff
@@ -37,7 +37,7 @@ module.exports = function postcssPlugin(snowpackConfig, options) {
 
       const encodedResult = await worker.transformAsync(contents, {
         config,
-        filepath: id,
```

Comment:
`id` is used in other parts of this file, and I believe those should stay as-is. This is the only line that causes errors, because `filepath` needs to point to a real file on disk